projects/plugins/jetpack/extensions/blocks/one-payment/variations.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * The different payment blocks that can be chosen.
+ */
+
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+const variations = [
```

Comment:
These should copy the title, icon etc from the inner blocks they have. Ideally it would programmatically get them.

Assuming it can programatically get them, this could just be a list of block names that are being included in the introducer which can be looped over to get a definition matching the variations api

Reply:
I've done this, except dynamic js imports besides the inefficiency, they wouldn't work for us (webpack tries to grab a load of php files). So each block requires an import + listing in the array